web/src/components/task/task-sidebar/finish-button/finish-button.tsx

```diff
@@ -60,9 +61,12 @@ export const FinishButton: FC<TaskSidebarProps> = ({
             (isValidation && !touchedPagesCount && !editedPagesCount);
     }
 
-    const tooltipContent = !isDisabled
-        ? null
-        : createTooltip(jobType === 'extensive_coverage', notProcessedPages);
+    const tooltipContent = createTooltip({
+        isDisabled,
+        isExtCov: jobType !== 'extensive_coverage',
```

Comment:
I propose to use the full name of the variable instead of using short version here - it'll look more straightforward from my perspective.